tests/ui/components/navigation/navigationcomponent.js

```diff
@@ -14,7 +20,11 @@ beforeEach(() => {
   DOM.empty(bodyEl);
 
   // Create the container that our component will be injected into
-  DOM.append(bodyEl, DOM.createEl('div', { class: 'test-component' }));
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+  // Mock DOM.on and DOM.off because they will not work in this test environment
+  DOM.on = jest.fn();
+  DOM.off = jest.fn();
```

Comment:
how come DOM.on and off don't work here? I thought we use this in other tests, but haven't looked at this in a while

Reply:
So I ALSO ran into this issue (when trying to write DirectAnswer tests), and my guess is that the jest DOM is not being kept in sync with how a real DOM would behave, and causes these event listener registrations to blow up because they're expecting an element to be there.

Reply:
That would make sense as the error I'm getting when I don't disable DOM.on and DOM.off is: 'Error initializing component (Navigation) Caused By: '[object Window]' is not a valid selector (unknown)'